Bug: https://bugs.gentoo.org/686982

TabStripModelChange has a defaulted default constructor and a const data member
without a user-defined default constructor.  This leads to a bug:

error: defaulting this default constructor would delete it after
its first declaration

We declare the data member as non-const instead.

--- chrome/browser/ui/tabs/tab_strip_model_observer.h
+++ chrome/browser/ui/tabs/tab_strip_model_observer.h
@@ -103,7 +103,7 @@ class TabStripModelChange {
 
  private:
   const Type type_ = kSelectionOnly;
-  const std::vector<Delta> deltas_;
+  std::vector<Delta> deltas_;
 
   DISALLOW_COPY_AND_ASSIGN(TabStripModelChange);
 };
